apis/nucleus/src/components/NebulaApp.jsx

```diff
@@ -64,6 +65,10 @@ export default function boot({ app, context }) {
   element.setAttribute('data-app-id', app.id);
   document.body.appendChild(element);
 
+  if (context) {
+    unifyContraintsAndInteractions(context);
```

Comment:
This runs anytime the context updates, and aims to populate both `contraints` and `interactions` correctly. It is especially required for the new `edit` constraint as that should be `true` by default, all other could have stayed as undefined.

Reply:
Oh, of course, you are right. I was thinking of boot as being a react component or hook for some reason, but then I get it! 👍